.travis.yml

```diff
@@ -1,17 +1,49 @@
-dist: "xenial"
-
 language: python
-script: python setup.py test
 
-matrix:
+script:
+  - python setup.py test
+
+stages:
+  - lint
+  - test
+
+jobs:
   include:
-    - python: "2.7"
-    - python: "3.3"
+    # Linting
+    - stage: lint
+      name: "Check linting with flake8"
+      install: pip install -r requirements.dev.txt
+      script: find . -name \*.py -exec flake8 {} +
+    - stage: lint
+      name: "Check format with black"
+      install: pip install -r requirements.dev.txt
+      script: find . -name \*.py -exec black --quiet --diff {} +
```

Comment:
Could this be simplified as:

    script: black --quiet --diff pymarc

It doesn't look like black will exit non-zero, so when it runs across errors it will simply print out some diffs and the build will continue? Is this really serving any purpose?

Reply:
My mistake, forgot the `--check` flag, fixed by https://github.com/edsu/pymarc/pull/150/commits/dddca146e7565d850f28cd345c226323c0d9182c